content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md

```diff
@@ -98,28 +84,32 @@ The theme has been moved outside of the Core repository since version 8.0.0.
 
 When a new release is built, this tag is needed to lock the theme version.
 
-- [Tag][git-tag] the new version:
+* [Tag][git-tag] the new version:
+
     ```bash
     git tag 2.0.0-rc1 # replace with your version
     ```
-- Push the tag:
+
+* Push the tag:
```

Comment:
@jolelievre pushing the tag should be the step **after** or **aligned* with the communication to prevent the situation where people are sharing the news about the new version being available, whereas the release process is still in progress

Reply:
Ah ok, then feel free to adapt this part to make sure it matches the need of the communication team

Reply:
That's why I didin't merge, I wanted to double check with you first